src/mame/palm/palm.cpp

```diff
@@ -16,195 +16,442 @@
 #include "machine/mc68328.h"
 #include "machine/ram.h"
 #include "sound/dac.h"
+#include "video/sed1375.h"
+
 #include "emupal.h"
 #include "screen.h"
 #include "speaker.h"
 
 #include "pilot1k.lh"
 
+#define VERBOSE (1)
+#include "logmacro.h"
```

Comment:
Do you want to go with verbose logging on in this driver for this release, or is this a leftover from debugging?